src/forms/validation-rules.js

```diff
@@ -203,16 +203,16 @@ export function validCollectedVariables(
   if (type === SIMPLE && value[0] || type === TABLE && value[0] ) {
     const typevalue = value[0].type;
     const typeexpectedVariables = expectedVariables[0].type;
-    if ((
+    if (
       value[0] &&
       value[0].codeListReference &&
-      value[0].codeListReference !== 'undefined')  || 
+      value[0].codeListReference !== 'undefined'  || 
       value[0] &&
       typevalue !== typeexpectedVariables || 
-      ( value[0] && 
-        typevalue === typeexpectedVariables &&
-        value[0][typevalue] !== expectedVariables[0][typeexpectedVariables]
-        )
+      value[0] && 
+      typevalue === typeexpectedVariables &&
+      JSON.stringify(value[0][typevalue]) !== JSON.stringify(expectedVariables[0][typeexpectedVariables])
```

Comment:
Are you sure with this ? 
Are you sure JSON.stringify will generate exactly the same string, with the same order ? 
![image](https://user-images.githubusercontent.com/555768/71259646-31924900-2339-11ea-8482-ceaa64823a5c.png)

Reply:
yes you are right sorry was using an other methods forget to take of the JSON.stringify